src/LaravelNovaCareersServiceProvider.php

```diff
@@ -34,6 +34,9 @@ public function boot()
         // Replace default controller so we can override the view that gets rendered.
         $this->app->bind(CareerController::class, NovaCareerController::class);
 
+        // Set TrafficCop on the CareerResource
+        CareerResource::$trafficCop = config('nova-careers.trafficCop');
```

Comment:
trafficCop should be using snake case lettering to keep it in line with other config variables.